locale/de/settings.cfg

```diff
@@ -1,37 +1,37 @@
 ltn-dispatcher-updates-per-tick=Anzahl der Stations- und Anforderungsaktualisierungen pro Tick.\nNiedrigere Werte verbessern Leistung.\nWird ignoriert wenn Update Frequenz > 1.
-ltn-dispatcher-requester-threshold=Mindestmenge an Artikeln um eine Lieferung auszulösen. \nKann durch Signal pro Anforderer-Haltestelle überschrieben werden.\ndefault = 1000
-ltn-dispatcher-provider-threshold=Mindestmenge an Artikeln um als Anbieter erkannt zu werden. \nKann durch Signal pro Anbieter-Haltestelle überschrieben werden.\ndefault = 1000
-ltn-dispatcher-schedule-circuit-control=Aktiviert Schaltungsbedinungen rot = 0 ODER grün ≥ 1 for alle Haltestellen.\nAchtung: Alle LTN Haltestellen müssen "an Zug senden" aktiviert und eine Schaltnetz-Verbindung haben. Sonst warten Züge ewig auf ein Signal.\ndefault = false
-ltn-dispatcher-depot-inactivity(s)=Inaktivitätsdauer in Sekunden bevor Züge das Depot verlassen.\ndefault = 5s
-ltn-dispatcher-stop-timeout(s)=Dauer in Sekunden bevor Züge ihre Station verlassen müssen.\n0 schaltet diese Funktion aus.\ndefault = 120s
-ltn-dispatcher-delivery-timeout(s)=Lieferzeit in Sekunden bevor Züge als vermisst gelten.\ndefault = 600s (10min)
-ltn-dispatcher-requester-delivery-reset=Inaktiv: (default)\nLieferung und Fahrplan werden bei Ankunft im Depot zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen haben keine Auswirkung.\n\nAktiv:\nLieferung und Fahrplan werden bei verlassen der Anforderer-Haltestelle zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen löscht die Lieferung und setzt den Fahrplan zurück.
-ltn-dispatcher-finish-loading=Aktiv: (default)\nVerhindert verlassen von Haltestellen während Greifarme/Pumpen arbeiten.\n\nInaktiv:\nZüge verlassen Haltestellen unmittelbar nach erreichen ihrer gewünschten Beladung.\nGreifarme bleiben Gegenstände haltend stecken, Tankwaggons enthalten Restmengen.
-ltn-depot-reset-filters=Aktiv: (default)\nFilter und Stapelbegrenzung in Güterwagons werden im Depot gelöscht.
-ltn-stop-default-network=Netzwerk ID wenn kein Signal "Kodierte Netzwerk ID" existiert.
\ No newline at end of file
+ltn-dispatcher-requester-threshold="Mindestmenge an Artikeln um eine Lieferung auszulösen. \nKann durch Signal pro Anforderer-Haltestelle überschrieben werden.\nStandardwert = 1000"
+ltn-dispatcher-provider-threshold="Mindestmenge an Artikeln um als Anbieter erkannt zu werden. \nKann durch Signal pro Anbieter-Haltestelle überschrieben werden.\nStandardwert = 1000"
+ltn-dispatcher-schedule-circuit-control="Aktiviert Schaltungsbedinungen rot = 0 ODER grün ≥ 1 for alle Haltestellen.\nAchtung: Alle LTN Haltestellen müssen \"an Zug senden\" aktiviert und eine Schaltnetz-Verbindung haben. Sonst warten Züge ewig auf ein Signal.\nStandardwert = false"
+ltn-dispatcher-depot-inactivity(s)="Dauer in Sekunden, bevor Züge das Depot aufgrund von Inaktivität verlassen.\nStandardwert = 5"

```

Comment:
"Dauer in Sekunden, bevor Züge das Depot wegen Inaktivität verlassen.\nStandardwert = 5"